### PR TITLE
Fixes #39 GetParam API to use strong typing

### LINKED ISSUE #39: For GetQueryParam API consider using strong typing
I'm not sure of the reason why `input` is of type `interface{}`.  Better if we can use a stricter typing.

_Originally posted by @thomas-fossati in https://github.com/veraison/veraison/pull/37#discussion_r603392092_

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
Use strong type for GetQueryParam API instead of interface{}

### CODE REVIEW

File: plugins/arangodbendorsements/arangodb.go [hunk 1]
```diff
@@ -119,8 +119,8 @@ func (e *ArangoStore) IsInitialised() bool {
 }
 
 // GetQueryParam returns a suitable query parameter based on supplied input
-func (e *ArangoStore) GetQueryParam(input interface{}) (string, error) {
-	switch input.(CollectionType) {
+func (e *ArangoStore) GetQueryParam(input string) (string, error) {
```
Comment (thomas-fossati) [inline plugins/arangodbendorsements/arangodb.go:122]: maybe instead of `input`, which has no particular meaning, use `queryParamID` or similar?

Comment (yogeshbdeshpande) [reply plugins/arangodbendorsements/arangodb.go:122]: ID looks a little odd as it is a string, may be simple to name it as qParam ?

Comment (thomas-fossati) [reply plugins/arangodbendorsements/arangodb.go:122]: the question is: what is this `input` parameter?  is it a query parameter (qParam)? or is it a unique identifier for the query parameter (qParamID)?

Comment (yogeshbdeshpande) [reply plugins/arangodbendorsements/arangodb.go]: Resolved it by renaming input to qParamID

**Review by thomas-fossati — APPROVED** [review]

**Pull request merged**